explorer/ast/expression_category.h

```diff
@@ -18,6 +20,8 @@ enum class ExpressionCategory {
   Initializing,
 };
 
+auto ExprCategoryToString(ExpressionCategory cat) -> llvm::StringRef;
```

Comment:
```suggestion
auto ExpressionCategoryToString(ExpressionCategory cat) -> llvm::StringRef;
```

This is used in one place, let's just match names.

Reply:
Done